exonum-java-binding/pom.xml

```diff
@@ -354,6 +354,7 @@
           <artifactId>maven-surefire-plugin</artifactId>
           <version>2.22.1</version>
           <configuration>
+            <redirectTestOutputToFile>true</redirectTestOutputToFile>
```

Comment:
What does it bring to us? We do not use stdout in tests so often.

Reply:
We don't, the logger prints to stdout (see the docs)